src/common/config.js

```diff
@@ -24,6 +24,11 @@ const schema = Joi.object({
   GRANT_APPLICATION_APPROVED_TOPIC_ARN: Joi.string().optional(),
   AWS_REGION: Joi.string(),
   AWS_ENDPOINT_URL: Joi.string().uri().optional(),
+  CASE_STAGE_UPDATES_SQS_URL: Joi.string().uri().optional(),
```

Comment:
This needs adding to https://github.com/DEFRA/cdp-app-config or the deploy will fail